src/Microsoft.Tye.Core/Serialization/YamlExtensionHelper.cs

```diff
@@ -0,0 +1,48 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using YamlDotNet.RepresentationModel;
+
+namespace Tye.Serialization
+{
+    public static class YamlExtensionHelper
+    {
+        internal static void HandleExtensionsMapping(YamlSequenceNode yamlSequenceNode, List<Dictionary<string, object>> extensions)
```

Comment:
Can the class be internal instead of the method? It's pretty rare to want a public type with internal methods.